app/frontend/src/features/communities/events/CreateEventPage.test.tsx

```diff
@@ -75,3 +72,96 @@ describe("Create event page", () => {
     expect(screen.getByTestId("event-page")).toBeInTheDocument();
   });
 });
+
+function renderPageWithState(state?: { communityId: number }) {
+  const { wrapper } = getHookWrapperWithClient({
+    initialRouterEntries: [
+      {
+        pathname: "/",
+        state,
+      },
+    ],
+  });
+  render(
+    <Switch>
+      <Route exact path="/">
+        <CreateEventPage />
+      </Route>
+      <Route path={eventRoute}>
+        <h1 data-testid="event-page">Event page</h1>
+      </Route>
+    </Switch>,
+    { wrapper }
+  );
+}
+
+describe("Create event page (offline events)", () => {
+  beforeAll(() => {
+    server.listen();
+  });
+
+  afterAll(() => {
+    server.close();
+  });
+
+  beforeEach(() => {
+    createEventMock.mockResolvedValue(events[0]);
+  });
+
+  it("creates on offline event with no route state correctly", async () => {
+    renderPageWithState();
+
+    userEvent.type(screen.getByLabelText(TITLE), "Test event");
+    userEvent.type(screen.getByLabelText(LOCATION), "tes{enter}");
+    userEvent.click(
+      await screen.findByText("test city, test county, test country")
+    );
+    userEvent.type(screen.getByLabelText(EVENT_DETAILS), "sick social!");
+    userEvent.click(screen.getByRole("button", { name: CREATE }));
+
+    await waitFor(() => {
+      expect(createEventMock).toHaveBeenCalledTimes(1);
+    });
+
+    expect(createEventMock).toHaveBeenCalledWith({
+      isOnline: false,
+      lat: 2,
+      lng: 1,
+      address: "test city, test county, test country",
+      title: "Test event",
+      content: "sick social!",
+      photoKey: "",
+      startTime: expect.any(Date),
+      endTime: expect.any(Date),
```

Comment:
msw and fakeTimers don't work well together - advancing the timer was causing all sort of weird React warnings 😢  so the check here sadly the assertions here have to be more generic as I couldn't use fake timer here to lock the Date. At least the more specific check is done with the online event test above though, which is shared logic so I think this will do...